mantidimaging/core/data/geometry.py

```diff
@@ -40,13 +42,46 @@ def __init__(self,
 
     def set_geometry_from_cor_tilt(self, cor: ScalarCoR, tilt: float) -> None:
         """
-        Converts a centre of rotation (that uses MI conventions) to the CIL convention.
+        Set's the geometry's CIL-convention Offset and Angle from the MI-convention Centre-of-Rotation and Tilt.
 
-        :param cor: A ScalarCoR object defining the centre of rotation as a float value.
+        :param cor: A ScalarCoR object defining the MI-convention centre of rotation as a float value.
         :type cor: ScalarCoR
         :param tilt: A float value defining the tilt in degrees.
         :type tilt: float
         """
-        offset: float = (cor.value - self.config.panel.num_pixels[0] / 2) * self.config.panel.pixel_size[0]
+
+        pixel_x_midpoint = self.config.panel.num_pixels[0] / 2
+        pixel_y_midpoint = self.config.panel.num_pixels[1] / 2
+        pixel_size = self.config.panel.pixel_size[0]
+
+        tilt_radians = -radians(tilt)
+        midpoint_cor = cor.value + pixel_y_midpoint * tan(tilt_radians)
+
+        offset: float = (midpoint_cor - pixel_x_midpoint) * pixel_size
 
         self.set_centre_of_rotation(offset=offset, angle=-tilt, angle_units='degree')
+
+    @property
+    def tilt(self) -> float:
+        slope: float = self.get_centre_of_rotation()['angle'][0]
```

Comment:
@samtygier-stfc made a good point to me that this may benefit from being called tilt instead of slope as we use slope to represent how far the COR moves from one slice to the next in the GUI